mcu/stm32l0xx.S

```diff
@@ -190,6 +190,20 @@ Reset_Handler:
     str     r2, [r0, #RCC_IOPENR]
     tst     r4, r4
     beq     .L_start_boot
+
+#if ( (defined(DFU_VERIFY_CHECKSUM) && (DFU_VERIFY_CHECKSUM != _DISABLE) ) )
+
+    ldr     r1, =#_APP_START
+    ldr     r2, =__romend
+    subs    r2, r1
+    subs    r2, r2, #8
```

Comment:
As per conversation, the adjustment should use a define from the checksum routine to determine the length, this would allow the checksum to be replaced with SHA or some other more advanced routine.